public/icons/copy-icon.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's from https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/copyshop-m.svg , i left it there in case someone in the future wants to try using it :)

in the meanwhile, the emoji has been added.